pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/JobConfigConstants.java

```diff
@@ -52,4 +52,17 @@
 
   // The path to the record reader to be configured
   public static final String RECORD_READER_PATH = "record.reader.path";
+
+  public static final String ENABLE_PARTITIONING = "enable.partitioning";
+  public static final String ENABLE_SORTING = "enable.sorting";
+  public static final String ENABLE_RESIZING = "enable.resizing";
+
+  public static final String PARTITION_COLUMN = "partition.column";
+  public static final String NUMBER_OF_PARTITIONS = "number.partitions";
+  public static final String PARTITION_FUNCTION = "partition.function";
+
+  public static final String SORTED_COLUMN = "sorted.column";
+
+  public static final String MINIMUM_NUMBER_OF_OUTPUT_FILES = "minimum.number.output.files";
```

Comment:
maybe min.num.output.files?

and max.num.records below to shorten parameters

Reply:
Good idea. Updated.